RevenueCatUI/CustomerCenter/Views/NoSubscriptionsView.swift

```diff
@@ -44,10 +44,11 @@ struct NoSubscriptionsView: View {
 
     var body: some View {
         VStack {
-            Text(localization.commonLocalizedString(for: .noSubscriptionsFound))
+            Text(self.configuration.screens[.noActive]?.title ?? "No Subscriptions found")
```

Comment:
We should come back to this since we shouldn't need to fallback here. Also, we need to build the buttons below from the paths provided by the backend.